app/helpers/summaries_helper.rb

```diff
@@ -7,4 +7,40 @@ def scheduler_data_param(user_attributes)
       a << [k, ERB::Util.url_encode(v)].join('=')
     end.join('&')
   end
+
+  def link_to_schedule_once_visit(library:, text:, css_class: nil)
+    link_to(
+      text,
+      library_schedule_path_map[library],
+      role: 'button',
+      class: css_class,
+      data: { 'mylibrary-modal': 'trigger' }
+    )
+  end
+
+  def schedule_once_link_or_dropdown
+    return 'Not eligible during current phase of Research Restart Plan' if library_schedule_path_map.blank?
+
+    schedulable_libraries = library_schedule_path_map.keys
+
+    if schedulable_libraries.one?
+      return link_to_schedule_once_visit(
+        library: schedulable_libraries.first,
+        text: "🗓 Schedule access to #{library_name(schedulable_libraries.first)}",
```

Comment:
@jvine the 🗓 wasn't in the mockup on #557, but I added it here.  Let me know if I should remove it.